From 28264f1be952e1df92b5e0649fdff27645a0f736 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 26 Sep 2013 15:02:30 +0800
Subject: [PATCH 253/602] ENGR00280171 CTS:fix location cts test

This commit do:
-remove redundant micro variable.
-for evk, auto, no gps support in hardware, set to no will not add gps provider.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 arm2_6dq/BoardConfig.mk       | 1 -
 arm2_6sl/BoardConfig.mk       | 1 -
 evk_6sl/BoardConfig.mk        | 3 +--
 evk_6sl/required_hardware.xml | 2 --
 imx53_smd/BoardConfig.mk      | 1 -
 sabreauto_6q/BoardConfig.mk   | 3 +--
 sabresd_6dq/BoardConfig.mk    | 1 -
 7 files changed, 2 insertions(+), 10 deletions(-)

diff --git a/arm2_6dq/BoardConfig.mk b/arm2_6dq/BoardConfig.mk
index 3b7e567..2597fe8 100755
--- a/arm2_6dq/BoardConfig.mk
+++ b/arm2_6dq/BoardConfig.mk
@@ -55,7 +55,6 @@ endif
 BOARD_HAVE_VPU := true
 BOARD_MODEM_VENDOR := AMAZON
 
-BOARD_HAVE_HARDWARE_GPS := true
 USE_ATHR_GPS_HARDWARE := true
 USE_QEMU_GPS_HARDWARE := false
 
diff --git a/arm2_6sl/BoardConfig.mk b/arm2_6sl/BoardConfig.mk
index 26aa6e2..7a2b227 100755
--- a/arm2_6sl/BoardConfig.mk
+++ b/arm2_6sl/BoardConfig.mk
@@ -55,7 +55,6 @@ endif
 BOARD_HAVE_VPU := true
 BOARD_MODEM_VENDOR := AMAZON
 
-BOARD_HAVE_HARDWARE_GPS := true
 USE_ATHR_GPS_HARDWARE := true
 USE_QEMU_GPS_HARDWARE := false
 
diff --git a/evk_6sl/BoardConfig.mk b/evk_6sl/BoardConfig.mk
index 2fb5e59..ebf4e7b 100644
--- a/evk_6sl/BoardConfig.mk
+++ b/evk_6sl/BoardConfig.mk
@@ -53,8 +53,7 @@ endif
 
 BOARD_MODEM_VENDOR := AMAZON
 
-BOARD_HAVE_HARDWARE_GPS := true
-USE_ATHR_GPS_HARDWARE := true
+USE_ATHR_GPS_HARDWARE := false
 USE_QEMU_GPS_HARDWARE := false
 
 #for accelerator sensor, need to define sensor type here
diff --git a/evk_6sl/required_hardware.xml b/evk_6sl/required_hardware.xml
index 725095a..5513a1e 100644
--- a/evk_6sl/required_hardware.xml
+++ b/evk_6sl/required_hardware.xml
@@ -22,8 +22,6 @@
     <feature name="android.hardware.camera" />
     <feature name="android.hardware.camera.front" />
     <feature name="android.hardware.camera.flash" />
-    <feature name="android.hardware.location" />
-    <feature name="android.hardware.location.network" />
     <feature name="android.hardware.touchscreen" />
     <feature name="android.hardware.touchscreen.multitouch" />
     <feature name="android.hardware.touchscreen.multitouch.distinct" />
diff --git a/imx53_smd/BoardConfig.mk b/imx53_smd/BoardConfig.mk
index 7967cab..2575fcc 100755
--- a/imx53_smd/BoardConfig.mk
+++ b/imx53_smd/BoardConfig.mk
@@ -43,7 +43,6 @@ HAVE_FSL_IMX_GPU3D := true
 HAVE_FSL_IMX_IPU := true
 BOARD_MODEM_VENDOR := AMAZON
 
-BOARD_HAVE_HARDWARE_GPS := true
 USE_ATHR_GPS_HARDWARE := true
 USE_QEMU_GPS_HARDWARE := false
 
diff --git a/sabreauto_6q/BoardConfig.mk b/sabreauto_6q/BoardConfig.mk
index 3fd6f0b..e6bce36 100644
--- a/sabreauto_6q/BoardConfig.mk
+++ b/sabreauto_6q/BoardConfig.mk
@@ -58,8 +58,7 @@ endif
 
 BOARD_MODEM_VENDOR := AMAZON
 
-BOARD_HAVE_HARDWARE_GPS := true
-USE_ATHR_GPS_HARDWARE := true
+USE_ATHR_GPS_HARDWARE := false
 USE_QEMU_GPS_HARDWARE := false
 
 #for accelerator sensor, need to define sensor type here
diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 100be6c..2abd908 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -53,7 +53,6 @@ endif
 
 BOARD_MODEM_VENDOR := AMAZON
 
-BOARD_HAVE_HARDWARE_GPS := true
 USE_ATHR_GPS_HARDWARE := true
 USE_QEMU_GPS_HARDWARE := false
 
-- 
1.8.0

